influxql/parser_test.go

```diff
@@ -2812,3 +2806,11 @@ func mustParseDuration(s string) time.Duration {
 	}
 	return d
 }
+
+func Duration(v time.Duration) *time.Duration {
```

Comment:
I moved them to private functions although I think I'll move them to `pkg` in the future. I just didn't want to go through the effort right now.